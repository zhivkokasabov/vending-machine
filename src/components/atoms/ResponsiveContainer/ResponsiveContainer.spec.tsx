import { render } from "@testing-library/react";
import { ResponsiveContainer } from ".";

describe("ResponsiveContainer tests", () => {
  const renderComponent = () => render(
    <ResponsiveContainer className="className" />
  );

  it("should render", () => {
    const { asFragment } = renderComponent();

    expect(asFragment()).toMatchSnapshot();
  });
});
