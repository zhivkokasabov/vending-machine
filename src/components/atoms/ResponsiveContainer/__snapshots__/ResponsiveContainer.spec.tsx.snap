// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResponsiveContainer tests should render 1`] = `
<DocumentFragment>
  <div
    class="p-4 className mx-auto w-full max-w-[1240]"
  />
</DocumentFragment>
`;
